### DISCUSSION TITLE
nodeSelector does not work for postgresql StatefulSet.

### DISCUSSION THREAD

**Discussion by huxuan** [apache/airflow#20051] [General]
### Official Helm Chart version

1.3.0 (latest released)

### Apache Airflow version

2.2.1

### Kubernetes Version

Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.7", GitCommit:"1dd5338295409edcfff11505e7bb246f0d325d15", GitTreeState:"clean", BuildDate:"2021-01-13T13:15:20Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}

### Helm Chart configuration

```
nodeSelector:
  key1: value1
  key2: value2
```

### Docker Image customisations

N/A

### What happened

StatefulSet for postgresql does not respect the nodeSelector specified in values.yaml.

### What you expected to happen

The nodeSelector also applies to postgresql StatefulSet.

### How to reproduce

helm template apache-airflow/airflow -f values.yaml > install.yaml

### Anything else

NOTE for willingness to submit an PR, since I am not so familiar with helm chart, it would be better to have some guidance.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
Similarly as with the other issue, Postgresql node is just used for development purpose and is discouraged from use in "production environment. Converting it to a discussion in case someone would likel to argue why our "development built-in Postgresql" shoudl support configurability

**Comment by potiuk** [comment]
Related to #20050 20050
